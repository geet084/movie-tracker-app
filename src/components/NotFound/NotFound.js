import React from 'react';

const NotFound = () => {
  return (
    <h4 className='no-faves error-404'>404 Page Not Found</h4>
  )
}

export default NotFound;