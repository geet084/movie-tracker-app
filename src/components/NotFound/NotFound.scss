.error-404 {
  margin-top: 0;
  padding-top: 300px;
}