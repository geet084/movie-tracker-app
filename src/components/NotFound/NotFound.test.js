import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
  let wrapper;
  wrapper = shallow(<NotFound />);

  it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});