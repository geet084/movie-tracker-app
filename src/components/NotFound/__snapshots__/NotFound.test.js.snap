// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match the snapshot with all data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "404 Page Not Found",
      "className": "no-faves error-404",
    },
    "ref": null,
    "rendered": "404 Page Not Found",
    "type": "h4",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "404 Page Not Found",
        "className": "no-faves error-404",
      },
      "ref": null,
      "rendered": "404 Page Not Found",
      "type": "h4",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
